qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-bamboo.properties

```diff
@@ -1,8 +1,9 @@
-webapp.url=https://qa-refapp.openmrs.org/openmrs
+webapp.url=http://localhost:8080/openmrs
```

Comment:
Does this change mean that one needs to have a local instance before they can run the tests successfully?

Reply:
Lately in september , with intellisoft, we used to use dockerised instance to spin up the instance,however it wasnt reliable especially if a fix is made with in the current module. probably @kdaud . You could help us understand the big reason behind this.

Reply:
> Does this change mean that one needs to have a local instance before they can run the tests successfully?

no !!

Because the `test-bamboo.properties` file is storing settings for tests to use when running on ci(Bamboo). Properties that are governing running the tests locally are residing in _test.properties_ file whose default url points to [qa- instance](https://github.com/openmrs/openmrs-contrib-qaframework/blob/853dfdd339baac31738988d01f466d85724171f5/qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test.properties#L1) however, one can change the url to point to a local instance in case has one and opts to use it since it does not depend on the internet connection.

Reply:
> we used to use dockerised instance to spin up the instance,however it wasnt reliable especially if a fix is made with in the current module

With the shared experience on the subject has made me give a thought again if the dockerized instance may not be stable and this is not the direction we would like to take. Not  sure whether our dockerised instance is stable however, its the one we are [using](https://github.com/openmrs/openmrs-contrib-qaframework/blob/853dfdd339baac31738988d01f466d85724171f5/qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-local.properties#L1) to run tests in github action and has never disappointed us ever since it was switched in June 2021.

Reply:
> `test-bamboo.properties` file is storing settings for tests to use when running on ci(Bamboo)

Does bamboo have an instance running at localhost:8080?

Reply:
Then why did you change the bamboo settings to localhost:8080?

Reply:
> why did you change the bamboo settings to localhost:8080

I had thought localhost:8080 would work since have seen this being te default port when installing a local instance is  `8080` using SDK.

@dkayiwa does this mean the port for `localhost` for Bamboo is a different one?

Reply:
Do you have any instance on bamboo running on localhost?

Reply:
@dkayiwa am not sure whether bamboo runs on localhost, and I have looked around but can't find a documentation that talks about it.

As per this conversation seems like bamboo does not run on local host and so we won't switch test-bamboo.properties settings. 

I will then opt for the first alternative I attached on the ticket for creating random values on fields that are causing test failure.